instrumentation/jms-1.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jms/JmsMessageProducerInstrumentation.java

```diff
@@ -129,6 +127,7 @@ public static void stopSpan(
       }
       CallDepthThreadLocalMap.reset(MessageProducer.class);
 
+      scope.close();
```

Comment:
...aaaand one more scope leak detected 😄 
It was worth adding that one test case